pkg/controller/import-controller.go

```diff
@@ -358,6 +365,45 @@ func (r *ImportReconciler) reconcilePvc(pvc *corev1.PersistentVolumeClaim, log l
 	return reconcile.Result{}, nil
 }
 
+func (r *ImportReconciler) copyImportProxyConfigMap(pvc *corev1.PersistentVolumeClaim, pod *corev1.Pod) error {
+	cdiConfig := &cdiv1.CDIConfig{}
+	if err := r.client.Get(context.TODO(), types.NamespacedName{Name: common.ConfigName}, cdiConfig); err != nil {
+		return err
+	}
+	cmName, err := GetImportProxyConfig(cdiConfig, common.ImportProxyConfigMapName)
+	if err != nil || cmName == "" {
+		return nil
+	}
+	cdiConfigMap := &corev1.ConfigMap{}
+	if err := r.uncachedClient.Get(context.TODO(), types.NamespacedName{Name: cmName, Namespace: r.cdiNamespace}, cdiConfigMap); err != nil {
+		return err
+	}
+	importConfigMap := &corev1.ConfigMap{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      GetImportProxyConfigMapName(pvc.Name),
+			Namespace: pvc.Namespace,
+			OwnerReferences: []metav1.OwnerReference{{
+				APIVersion:         pod.APIVersion,
+				Kind:               pod.Kind,
+				Name:               pod.Name,
+				UID:                pod.UID,
+				BlockOwnerDeletion: pointer.Bool(true),
+				Controller:         pointer.Bool(true),
+			}},
+		},
+		Data: cdiConfigMap.Data,
+	}
+	if err := r.client.Create(context.TODO(), importConfigMap); err != nil && !k8serrors.IsAlreadyExists(err) {
+		return err
+	}
+	return nil
+}
+
+// GetImportProxyConfigMapName returns the import proxy ConfigMap name
+func GetImportProxyConfigMapName(pvcName string) string {
```

Comment:
I think there is a slim chance this will re-introduce some resource name-length problems
so we might have to use `naming.GetResourceName` and friends here

Reply:
fixed